DisCatSharp.ApplicationCommands/ApplicationCommandsExtension.cs

```diff
@@ -2155,4 +2237,220 @@ internal class ContextMenuCommand
         /// </summary>
         public MethodInfo Method { get; set; }
     }
+
+    #region Default Help
+    /// <summary>
+    /// Represents the default help module.
+    /// </summary>
+    public class DefaultHelpModule : ApplicationCommandsModule
+    {
+        public class DefaultHelpAutoCompleteProvider : IAutocompleteProvider
+        {
+            public async Task<IEnumerable<DiscordApplicationCommandAutocompleteChoice>> Provider(AutocompleteContext context)
+            {
+                var options = new List<DiscordApplicationCommandAutocompleteChoice>();
+
+                var globalCommands = await context.Client.GetGlobalApplicationCommandsAsync();
+                var guildCommands = await context.Client.GetGuildApplicationCommandsAsync(context.Guild.Id);
+                var slashCommands = globalCommands.Concat(guildCommands)
+                .Where(ac => !ac.Name.Equals("help", StringComparison.OrdinalIgnoreCase))
+                .GroupBy(ac => ac.Name).Select(x => x.First());
+                var list = slashCommands.ToList();
+                var count = 0;
+                foreach (var sc in list)
```

Comment:
@Lulalaby I believe 24 is our maximum for this right?

Otherwise, we could do a list.Take(24) in this to avoid the need for counter.  Even if your list is 10 items it would simply provide the 10 items. However if 50, it'd grab the first 24 items in sequence.

Reply:
O.o didn't knew about this function.
Max is always 25

Reply:
Gotcha I thought so but wasn't sure

Reply:
I start counting at 0 XD

Reply:
@Lulalaby I realized I skipped one of your comments lol

The Take and Skip functions are pretty awesome for pagination purposes.  

We're all familiar with the idea of viewing X amount of items at a time.... 20, 50, 100, whatever.
You can accomplish that by doing the following -- assuming page counter starts at 0
```csharp
var pageResults = list.Skip(page * pageSize).Take(pageSize).ToList();
```

Reply:
Oi

That's nice

Reply:
> 

hi